httpx/_transports/default.py

```diff
@@ -0,0 +1,131 @@
+"""
+Custom transports, with nicely configured defaults.
+
+The following additional keyword arguments are currently supported by httpcore...
+
+* uds: str
+* local_address: str
+* retries: int
+* backend: str ("auto", "asyncio", "trio", "curio", "anyio", "sync")
+
+Example usages...
+
+# Disable HTTP/2 on a single specfic domain.
```

Comment:
This kind of super granular usage enabled by mounts is _really_ cool. :-)

Reply:
Indeed!
I've added this one to the docs too, now.